pkg/agent/proxy/proxier.go

```diff
@@ -98,12 +110,7 @@ func (p *proxier) removeStaleServices() {
 			klog.Errorf("Failed to remove flows of Service %v: %v", svcPortName, err)
 			continue
 		}
-		for _, endpoint := range p.endpointsMap[svcPortName] {
-			if err := p.ofClient.UninstallEndpointFlows(svcInfo.OFProtocol, endpoint); err != nil {
-				klog.Errorf("Failed to remove flows of Service Endpoints %v: %v", svcPortName, err)
-				continue
-			}
-		}
+		// We don't need to uninstall Endpoint flows,
```

Comment:
I think you can remove this line, the function comment is good enough

Reply:
Removed.